source/functions/handle-errors.txt

```diff
@@ -0,0 +1,422 @@
+
+      You must include the correct parameters to make this function work
+      with the rest of the retry logic. These parameters are:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50
+
+         * - Parameter
+           - Description
+
+         * - ``...args``
+           - Zero or more parameters to pass to the function with main logic.
+             In the case of this example, the two numbers added
+             in ``additionOrFailure()``, ``num1`` and ``num2``.
+
+         * - ``operationId``
+           - :ref:`BSON.Object.Id<bson-objectid>`. Unique identifier for the Function
+             call and retries. Set default value to a ``new BSON.ObjectId()``.
+
+         * - ``retries``
+           - Number. Set default value to 0.
+
+      The body of ``additionWithRetryHandler`` is the retry handler ``handleRetry``
+      invoked by ``context.functions.execute()``,
+      which in turn invokes ``additionOrFailure``. The arguments you pass to
+      context.functions.execute() are the following:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50
+
+         * - Argument
+           - Description
+
+         * - ``"handleRetry"``
+           - String. Name of the Function you defined to invoke the main function
+             and post to the retry logs if the main function doesn't properly execute.
+
+         * - ``additionOrFailure``
+           - JavaScript function. The main function that ``handleRetry()`` invokes.
+
+         * - ``operationId``
+           - BSON.ObjectId. Passed in as argument from the parameter ``operationId`` of ``additionWithRetryHandler()``.
+
+         * - ``retries``
+           - Number. Passed in as argument from the parameter ``retries`` of ``additionWithRetryHandler()``.
+
+         * - ``...args``
+           - Zero or more arguments to pass to the function with main logic.
+             Passed in as argument from the parameter ``...args`` of ``additionWithRetryHandler()``
+
+      .. tabs-realm-admin-interfaces::
+
+         .. tab::
+            :tabid: cli
+
+            Add the Function metadata to :file:`functions/config.json`:
+
+            .. code-block:: js
+               :caption: functions/config.json
+
+               [
+                  // ...other configuration
+                  {
+                    "name": "additionWithRetryHandler",
+                    "private": false
+                  }
+               ]
+
+            Add the following code to the file :file:`functions/additionWithRetryHandler.js`:
+
+            .. literalinclude:: /examples/generated/functions/additionWithRetryHandler.snippet.addition-with-retry-handler.js
+              :language: js
+              :caption: functions/additionWithRetryHandler.js
+
+         .. tab::
+            :tabid: ui
+
+            In the field :guilabel:`Function Name`, add ``additionWithRetryHandler``.
+
+            For the field :guilabel:`Function`, add the following code:
```

Comment:
In Step 1 above, we say "add the following code, then save the Function." We don't say to save the function in step 2 or 3 here, and we don't say in any of the CLI steps to push the updated config files. We should probably be consistent in all of these places, and I think I'd err on the side of being explicit about saving/pushing just to cover that base. (And maybe link to the CLI push documentation for the CLI steps?)